pkg/discoverer/k8s/router/router.go

```diff
@@ -58,12 +58,20 @@ func New(opts ...Option) http.Handler {
 				h.Index,
 			},
 			{
-				"Discover",
+				"Pods",
 				[]string{
 					http.MethodPost,
```

Comment:
![badge](https://img.shields.io/badge/review-must-red.svg?style=flat-square)
It is not consistent with apis/proto/discoverer/discoverer.proto. it should be MethodGet.